tests/e2e/cht-form/default/death-report.wdio-spec.js

```diff
@@ -0,0 +1,59 @@
+const mockConfig = require('../mock-config');
+const moment = require('moment/moment');
+const deathReportForm = require('@page-objects/default/enketo/death-report.page');
+const genericForm = require('@page-objects/default/enketo/generic-form.wdio.page');
+
+describe('cht-form web component - Death Report Form', () => {
+
+  it('should submit a death report', async () => {
+    await mockConfig.startMockApp('default', 'app', 'death_report');
+
+    await browser.execute(() => {
+      const myForm = document.getElementById('myform');
+      myForm.content = { contact: { _id: '12345'} };
+    });
+
+    const date = moment().format('YYYY-MM-DD');
+    const deathNote = 'Test note';
+    const title  = await genericForm.getFormTitle();
+    expect(title).to.equal('Death report');
+
+    await deathReportForm.selectDeathPlace(deathReportForm.PLACE_OF_DEATH.healthFacility);
+    await deathReportForm.setDeathInformation(deathNote);
+    await deathReportForm.setDeathDate(date);
+    await genericForm.nextPage();
+
+    const {deathDate, deathInformation} = await deathReportForm.getSummaryDetails();
+    expect(deathDate).to.equal(date);
+    expect(deathInformation).to.equal(deathNote);
+
+    const data = await mockConfig.submitForm();
+    const jsonObj = data[0].fields.death_details;
```

Comment:
```suggestion
    const [doc, ...additionalDocs] = await mockConfig.submitForm();
    expect(additionalDocs).to.be.empty;
    const jsonObj = doc.fields.death_details;
```

These tests are well positioned to be able to assert exactly what docs are created by a form (something that is more challenging in a regular e2e test).  Could you possibly update similar code in tests like this to assert that only one doc is created?

Reply:
Sure, that is a great idea.
Done ⭐